app/assets/stylesheets/application.scss

```diff
@@ -30,7 +30,18 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .post-it {
+ background:#fefabc;
+ padding:15px;
```

Comment:
Properties should be ordered background, -moz-box-shadow, -webkit-box-shadow, box-shadow, color, font-size, padding, width<br>Colon after property should be followed by one space